### PR TITLE
Network Type added in wallet template

### PR DESCRIPTION
**Pull request by rmeena840**
Task: https://www.pivotaltracker.com/story/show/181830978

Screenshot 2022-04-22 at 12 53 14 PM

### CODE REVIEW

File: wallet/src/App.vue [hunk 1]
```diff
@@ -80,27 +110,37 @@ export default defineComponent({
...
-          this.connectWalletConnect();
-          break;
-        default:
-          console.warn("Wallet %s not supported", e.target.value);
+      if (e.target.value) {
+        let walletType = e.target.value;
+        switch (walletType) {
+          case WalletType.ALGOSIGNER:
+            this.connectAlgoSigner();
+            break;
```
Comment (vuvoth) [inline wallet/src/App.vue]:
```suggestion
          case WalletType.ALGOSIGNER: {
               this.connectAlgoSigner();
               break;
            }
```

Comment (vuvoth) [reply wallet/src/App.vue]: I think it more sense. However you also can skip this.

File: wallet/src/App.vue [hunk 2]
```diff
@@ -47,12 +59,12 @@ import {
 } from "@algo-builder/web";
 import { convertMicroAlgoToAlgo } from "./utility";
 declare var AlgoSigner: any; // eslint-disable-line
```
Comment (vuvoth) [inline wallet/src/App.vue:61]: Could we use [algo-signer-type](https://github.com/scale-it/algo-builder/blob/develop/packages/web/src/algo-signer-types.ts) for AlgoSigner?

**Review by vuvoth — COMMENTED** [review]
left few comments. thanks

File: wallet/src/App.vue (hunk 2 above)
Comment (rmeena840) [reply wallet/src/App.vue:61]: No. We can't. When you have the AlgoSigner extension installed, then the AlgoSigner object is present in the windows object. I have also checked with the AlgoSigner community. It has to be `Any`. This is also described in their documentation as well.

File: wallet/src/App.vue (hunk 1 above)
Comment (rmeena840) [reply wallet/src/App.vue]: Done

File: wallet/src/App.vue (hunk 2 above)
Comment (vuvoth) [reply wallet/src/App.vue:61]: Okay ~~ thank!

**Review by Megha-Dev-19 — APPROVED** [review]
Lgtm, Thanks for the PR

**Pull request merged**